lambda/src/main/resources/db/migration/V21__add-consignment-seq-and-reference.sql

```diff
@@ -0,0 +1,11 @@
+-- Create sequence that will be the consignment sequence ID used to encode b25 section of consignment reference
+CREATE SEQUENCE "ConsignmentSequenceID"
```

Comment:
Did you manage to get a sequence working in the H2 test database?

Reply:
I did yeah! Works almost exactly the same as the real DB sequence :tada: